CredentialCardField.vue

```diff
@@ -218,8 +220,14 @@ export default {
     word-wrap: break-word;
   }
 
-  .s-toggle-icon {
-    font-size: 20px;
+  .s-toggle {
+    width: 38px;
+
+    .s-toggle-icon {
+      font-size: 20px;
+    }
   }
+
+
```

Comment:
These extra whitespaces must not be getting picked up by the linter ... can you remove these?